modules/issue/template/template.go

```diff
@@ -172,10 +178,38 @@ func validateOptions(field *api.IssueFormField, idx int) error {
 				return position.Errorf("'label' is required and should be a string")
 			}
 
+			if visible, ok := opt["visible"]; ok {
```

Comment:
If this is for all types, shouldn't this be checked for all types as well?

Reply:
I dont get your question ... it only works that way ...

Reply:
Thats because we use no simple boolean options but slice of string ... witch are deserialized to `interface{}` ...

Reply:
For all types there is the visibility yes. But checkboxes also can have it per checkbox